packages/stark-core/src/modules/routing/services/routing.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Action, Store } from "@ngrx/store";
-import { Observable } from "rxjs/Observable";
-import { fromPromise } from "rxjs/observable/fromPromise";
-import { empty } from "rxjs/observable/empty";
+import { Observable, empty } from "rxjs";
+// FIXME Is fromPromise still valid ?
+import { fromPromise } from "rxjs/internal/observable/fromPromise";
```

Comment:
I think it is now the "from" observable that includes the promises, arrays, scalar values: https://github.com/ReactiveX/rxjs/blob/master/MIGRATION.md#observable-classes

You could give it a try ;)

Reply:
that seems to work :)